.cpp

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+Int main()
+{
+    Int m1,m2,m3,m4,m5,per;
+    Printf("enter marks in five subjects");
+    Scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
+
+     Per=(m1+m2+m3+m4+m5)/500*100;
+     
+    If(per>=60)
```

Comment:
Is this code working? You have Per in line 8, and at rest of the places its per.
Could you please confirm?